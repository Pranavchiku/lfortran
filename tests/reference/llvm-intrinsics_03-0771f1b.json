{
    "basename": "llvm-intrinsics_03-0771f1b",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_03.f90",
    "infile_hash": "8e3eaf17fa2610e5ebdf1fd94dbf2be8ff26f85160239bcec926e16f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-intrinsics_03-0771f1b.stdout",
    "stdout_hash": "2e3fff65e7077427426e022a71ad53587f9a21b5f3ec38c3d91166a5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}