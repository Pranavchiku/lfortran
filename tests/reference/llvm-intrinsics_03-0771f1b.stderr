warning: Implied save feature
 --> tests/../integration_tests/intrinsics_03.f90:3:1
  |
3 | real :: x
  | ^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/intrinsics_03.f90:4:1
  |
4 | real(dp) :: a, r1, r2
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/intrinsics_03.f90:4:1
  |
4 | real(dp) :: a, r1, r2
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/intrinsics_03.f90:4:1
  |
4 | real(dp) :: a, r1, r2
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
